airflow/providers/google/cloud/triggers/kubernetes_engine.py

```diff
@@ -18,14 +18,108 @@
 from __future__ import annotations
 
 import asyncio
+from datetime import datetime
 from typing import Any, AsyncIterator, Sequence
 
 from google.cloud.container_v1.types import Operation
 
-from airflow.providers.google.cloud.hooks.kubernetes_engine import AsyncGKEHook
+try:
+    from airflow.providers.cncf.kubernetes.triggers.pod import KubernetesPodTrigger
+except ImportError:
+    # preserve backward compatibility for older versions of cncf.kubernetes provider
+    from airflow.providers.cncf.kubernetes.triggers.kubernetes_pod import KubernetesPodTrigger
+from airflow.providers.google.cloud.hooks.kubernetes_engine import AsyncGKEHook, AsyncGKEPodHook
 from airflow.triggers.base import BaseTrigger, TriggerEvent
 
 
+class GKEPodTrigger(KubernetesPodTrigger):
```

Comment:
shouldnt this be `GKEStartPodTrigger` instead of `GKEPodTrigger` - its being used only by `GKEStartPodOperator`.

Reply:
Done